### DISCUSSION TITLE
Custom CSS also for <h1>

### DISCUSSION THREAD

**Discussion by TorstenC** [docToolchain/docToolchain#1455] [Q&A]
Hallo,

wenn man einen pass-through block für `<style>` verwendet wie in [Working with the HTML style](https://doctoolchain.org/docToolchain/v2.0.x/020_tutorial/030_generateHTML.html#_working_with_the_html_style) beschrieben, muss er nach der Überschrift kommen, wird hinter `<div class="sectionbody">` gerendert und schließt die `<h1>Seitenüberschrift</h1>` aus.

Alternativ mit Fehlermeldung vom AsciiDoc-Linter geht auch davor, also direkt nach `<div id="content">`:

![image](https://github.com/user-attachments/assets/38bf25c3-131a-4c6e-a4d7-7fe95ac37c10)

- Wie geht Ihr damit um?
- Gibt es noch einen Trick?

BTW, @rdmueller: Du Schreibst bei [Help wanted #607](https://github.com/docToolchain/docToolchain/discussions/607): _"We need people who create a nice theme for the new static site generator."_  
Ich bin aktuell noch beim Anpassen an unsere CI-Farben _([included-css](https://gist.github.com/TorstenC/ca419b18d51d1ce92b107221771af863))_.  
Wenn ich das alles verstanden habe und sich danach meine Prioritäten nicht verschieben, würden mich _'microsites'_ (mit `jBake` site generator) auch interessieren, mit Light- und Dark-Mode zum Umschalten. 😉

**Comment by TorstenC** [comment]
PS:  
I changed to `:doctype: book` and made my Level-0-headline to `[section]` to get rid of this AsciiDoc-Linter-Error in VS Code.

**Comment by rdmueller** [comment]
Hallo Torsten,

inline Changes am CSS zu machen ist natürlich recht einfach. Hast Du schon mal darüber nachgedacht über das Theme zu gehen?

https://doctoolchain.org/docToolchain/v2.0.x/020_tutorial/040_microsite/130_theming.html

Du kopierst mit `copyThemes` erst das Theme in Dein Projekt und kannst dann alle Templates ändern. Alles, was Du nicht angefasst hast, kannst Du wieder löschen.
